test/unit/runtime/Loader.js

```diff
@@ -230,6 +237,7 @@ suite('Loader.js', function() {
   });
 
   test('LoaderImport.Fail.deperror', function(done) {
+    this.timeout(4000);
```

Comment:
It takes 1s to process the `catch` block in `UncoatedModuleInstantiator.prototype.getUncoatedModule`.  The individual parts of the block seem to take no time on the ms scale. As far as I can tell so far this has to be a V8 issue.